bootstraptest/test_yjit.rb

```diff
@@ -5015,13 +5015,14 @@ def test_body(args) = Array(1, *args)
 }
 
 # compiling code shouldn't emit warnings as it may call into more Ruby code
-assert_normal_exit %q{
+assert_equal 'ok', <<~'RUBY'
   # [Bug #20522]
+  $VERBOSE = true
   Warning[:performance] = true
 
   module StrictWarnings
     def warn(msg, category: nil, **)
-      raise msg
+      raise warn
```

Comment:
Not sure if I follow 🤔 Is `warn` a method call (of itself)?

Reply:
I was also confuzzled

Reply:
https://github.com/ruby/ruby/pull/10912/files#r1626489542

Reply:
> bootstraptest.test_yjit.rb_5020_1.rb:8:in 'warn': wrong number of arguments (given 0, expected 1) (ArgumentError)